src/containers/apply/DeveloperInfo.tsx

```diff
@@ -17,75 +17,65 @@ interface IDeveloperInfoProps {
   updateOrganization: (value: IErrorableInput) => void;
 }
 
-const mapStateToProps = (state: IRootState) => {
-  return {
-    email: state.application.inputs.email,
-    firstName: state.application.inputs.firstName,
-    lastName: state.application.inputs.lastName,
-    organization: state.application.inputs.organization,
-  };
-};
+const mapStateToProps = (state: IRootState) => ({
+  email: state.application.inputs.email,
+  firstName: state.application.inputs.firstName,
+  lastName: state.application.inputs.lastName,
+  organization: state.application.inputs.organization,
+});
 
 type DeveloperInfoDispatch = ThunkDispatch<
-  IRootState, 
-  undefined, 
-  actions.UpdateApplicationAction
+IRootState, 
+undefined, 
+actions.UpdateApplicationAction
```

Comment:
I could be going against the linter here, but, again it seems like these should be indented?
```suggestion
  IRootState,
  undefined,
  actions.UpdateApplicationAction
```

Reply:
It is in fact a tslint enforced rule. I agree it looks a little weird.

Reply:
The TS linter does want a no indent here.